src/main/scala/zio/codec/Equiv.scala

```diff
@@ -3,9 +3,18 @@ package zio.codec
 import zio.Chunk
 
 sealed trait Equiv[A, B]
+
 object Equiv {
   final case class Left[A, B](b: B)  extends Equiv[(A, B), A]
   final case class Right[A, B](a: A) extends Equiv[(A, B), B]
 
-  final case object UInt16 extends Equiv[Chunk[Boolean], Int]
+  final case class Ignore[A, Unit](a: A) extends Equiv[A, Unit]
```

Comment:
It's possible it may make sense to move `Equiv` into `CodecModule`, so that different modules can specialize it. 

But only if it turns out to be painful to support all `Equiv` inside all modules.